specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2020-04-01-preview/resourcechanges.json

```diff
@@ -0,0 +1,405 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Resource Graph",
+    "description": "Azure Resource Graph API Reference",
+    "version": "2020-04-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.ResourceGraph/resourceChanges": {
+      "post": {
+        "description": "List changes to a resource for a given time interval.",
+        "operationId": "ResourceChanges",
```

Comment:
OperationId should be in Noun_Verb format. Suggest `ResourceChanges_List`.  If you have/plan additional List at different scope either now or later, you could add a `ByXXX` suffix here.  Such as `ResourceChanges_ListByTenant` as this is across all subs.

Reply:
Thanks for the suggestions. For now we'd like to keep it as-is, but we'll definitely consider revisiting the operationId in future releases.